src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="action"
+      method="get"
+      data-qa="big"
+      >
+      <label for="big_Search">
+        <input
+          type="search"
+          name="bigSearch"
+          placeholder="Try &quot;Los Angeles&quot"
+          id="big_Search"
+          class="search bigSearch"
+          data-qa="keypress"
+          >
+        <img
+        class="bigIcon"
+        src="images/Search.svg"
+        alt="bigIcon"
+        >
```

Comment:
check the checklis file from Readme, hint №1
```suggestion
    <form
      action="action"
      method="get"
      data-qa="big"
    >
      <label for="big_Search">
        <input
          type="search"
          name="bigSearch"
          placeholder="Try &quot;Los Angeles&quot"
          id="big_Search"
          class="search bigSearch"
          data-qa="keypress"
        >
        <img
          class="bigIcon"
          src="images/Search.svg"
          alt="bigIcon"
        >
```